Frameworks/VMFleet/VMFleet.psm1

```diff
@@ -8665,4 +9136,152 @@ function SetCacheBehavior
             Remove-PSSession -Session $s
         }
     }
+}
+
+function Get-Salt {
+    $salt = ([char]'a'..[char]'z' + [char]'A'..[char]'Z' + [char]'0'..[char]'9' | Get-Random -Count 4 |% { [char]$_ }) -join ''
+    Write-Host "Generated Salt: $salt"
```

Comment:
does this ever need to be output in normal practice? consider verbose if you think its really needed.